Don't force the use of cc when compiling a test shared object; get
that passed in from the makefile

Not intended for upstream.

--- cffi-2.0.0/testing/cffi0/test_ownlib.py
+++ cffi-2.0.0/testing/cffi0/test_ownlib.py
@@ -175,7 +175,7 @@ class TestOwnLib(object):
                 unicode_name = u+'testownlib'
                 encoded = str(unicode_name)
             subprocess.check_call(
-                "cc testownlib.c -shared -fPIC -o '%s.so'" % (encoded,),
+                os.getenv('TESTOWNLIB_CC') % ('testownlib.c', encoded),
                 cwd=str(udir), shell=True)
             cls.module = os.path.join(str(udir), unicode_name + (u+'.so'))
         print(repr(cls.module))
